### DISCUSSION TITLE
[Feature Request] Kanban Board

### DISCUSSION THREAD

**Discussion by edfloreshz** [heroui-inc/heroui#4721] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently there’s no Kanban component.

### Describe the solution you'd like

It would be awesome to have a built in component to build Kanban boards.

### Describe alternatives you've considered

https://www.npmjs.com/package/@syncfusion/ej2-react-kanban

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1754/[feature-request]-kanban-board">ENG-1754 [Feature Request] Kanban Board</a></p>

**Comment by ssijak** [comment]
Why would there be such a complex component in a base UI kit?
